sentry-types/src/protocol/attachment.rs

```diff
@@ -0,0 +1,80 @@
+use std::fmt;
+
+/// The different types an attachment can have
```

Comment:
```suggestion
/// The different types an attachment can have.
```

Please make sure that all the doc comments have trailing periods ;-)